### ISSUE TITLE
Spectralink devices never show Earpiece as an option

### ISSUE THREAD

**Issue by ysaber** [twilio/audioswitch#170]
**Description**
[Spectralink](https://www.spectralink.com/) devices never show "Earpiece" as an option for audio devices when on a video call.

**Steps to reproduce**
1. On any Spectralink device, start a video call using the Twilio API
2. Check available audio devices using audioSwitch.availableAudioDevices

**Expected behavior**
As 1 of 2 _always_ available audio devices, Earpiece should _always_ be listed 

**Actual behavior**
Only Speakerphone is listed

- AudioSwitch Version: 1.2.0

**Android Device (please complete the following information):**
 - Device: Any [Spectralink](https://www.spectralink.com/) device
 - API Version: Various

**Screenshots**
![speaker](https://github.com/user-attachments/assets/11f0cb9e-7491-4bfd-a592-5ed97921d91b)

**Notes**
All other OEMs show Earpiece and Speakerphone as the default audio devices.

**Comment by ocarevs** [comment]
Hi @ysaber thank you for reporting the issue. We're looking in to this. I assume this is happening specifically on Spectral link devices?

**Comment by ysaber** [comment]
@ocarevs correct, this issue is not reproducible on any other device (we tried Pixels, Samsungs, Zebras, and a few others). We have, however, reproduced it on a number of Spectralink devices (Versity 92, Versity 95, etc).

**Comment by ysaber** [comment]
Hi @ocarevs , do you have any updates on this issue?

**Comment by ocarevs** [comment]
Hi @ysaber we currently don't have access to Spectralink device, I'll let you know if we have any updates on this issue.

**Comment by afalls-twilio** [comment]
Closing due to inactivity
